spd/configs.py

```diff
@@ -153,7 +154,8 @@ class IHTaskConfig(BaseModel):
         description="Number of tokens to use as a prefix window for the induction head",
     )
 
-TaskConfig: TypeAlias = TMSTaskConfig | ResidualMLPTaskConfig | LMTaskConfig | IHTaskConfig
+
+type TaskConfig = TMSTaskConfig | ResidualMLPTaskConfig | LMTaskConfig | IHTaskConfig
```

Comment:
I think you can just do:
```suggestion
TaskConfig = TMSTaskConfig | ResidualMLPTaskConfig | LMTaskConfig | IHTaskConfig
```

Though I don't mind being explicit that it's a type alias, maybe it's not consistent with the rest of the codebase though. So probably best to remove.